packages/core/nuxt-theme-module/theme/components/InstagramFeed.vue

```diff
@@ -34,6 +35,30 @@ export default {
   components: {
     SfSection,
     SfImage
+  },
+  setup() {
+    const isMobile = ref(false);
```

Comment:
Can't we just import mobile observers `isMobile` from sfui here?

Reply:
it doesn't work properly yet